### PR TITLE
Fix trunk; no default C++ operators on our version of C++

### PR DESCRIPTION
**Pull request by scotts**
PR #354 broke the trunk because it took advantage of default C++ operators, and that is a C++20 feature. We're not on C++20 yet. I'm not sure why the tests did not run on that PR. This PR should fix the trunk.

### CODE REVIEW

File: src/torchcodec/decoders/_core/VideoDecoder.cpp [hunk 1]
```diff
@@ -1340,7 +1352,11 @@ int VideoDecoder::convertFrameToBufferUsingSwsScale(
   int expectedOutputHeight = outputTensor.sizes()[0];
   int expectedOutputWidth = outputTensor.sizes()[1];
   auto curFrameSwsContextKey = SwsContextKey{
-      frame->width, frame->height, frameFormat, expectedOutputWidth, expectedOutputHeight};
+      frame->width,
+      frame->height,
+      frameFormat,
+      expectedOutputWidth,
+      expectedOutputHeight};
```
Comment (scotts) [inline src/torchcodec/decoders/_core/VideoDecoder.cpp:1359]: This is a drive-by linting. I think it's easier to keep it in the PR.

**Review by ahmadsharif1 — APPROVED** [review]
